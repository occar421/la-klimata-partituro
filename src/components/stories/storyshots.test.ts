// @ts-ignore
import initStoryshots from '@storybook/addon-storyshots'; // no .d.ts file

initStoryshots({
    /*
     configuration options
     */
});