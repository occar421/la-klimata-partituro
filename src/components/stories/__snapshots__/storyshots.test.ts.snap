// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Buttona with some emoji 1`] = `
<button
  className="button button-extra"
  onClick={[Function]}
>
  😀 😎 👍 💯
</button>
`;

exports[`Storyshots Buttona with text 1`] = `
<button
  className="button button-extra"
  onClick={[Function]}
>
  Hello Button!
</button>
`;
